ds/data/processed/industry/employment_high_tech.nuts2.yaml

```diff
@@ -0,0 +1,29 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: POST
+data_date: 20210601
+title: Employment in high tech industries
+subtitle: This indicator measures employment in high tech industries based on the Business Register Employment Survey. High tech industries are defined in terms of SIC-4 (industry) codes, using the definition in "The Geography of the UK's creative and high-tech economies" (https://media.nesta.org.uk/documents/geography_uks_creative_high-tech_economieswv20151.pdf) in table 6.1.3.4.  
```

Comment:
Sorry I just noticed that the subtitle is very long, I think you could shorten it and eventually provide the current text as the value of `description` (please check the template).

<img width="1339" alt="Screenshot 2021-06-02 at 12 43 39" src="https://user-images.githubusercontent.com/1309648/120474645-59943180-c3a0-11eb-9669-a55b2621a9e0.png">